network-protection/network-protection-subscription-internal/src/main/java/com/duckduckgo/networkprotection/subscription/ui/NetpVerifySubscriptionViewModel.kt

```diff
@@ -80,7 +80,7 @@ class NetpVerifySubscriptionViewModel @Inject constructor(
 
     private fun attemptToAuthorize() {
         viewModelScope.launch {
-            netpSubscriptionManager.authorize()
+            netpAuthManager.authorize()
         }
     }
```

Comment:
Do we need this method at all? we can just call that as part of `getState()` flow in `onStart() {}`

Reply:
Will not fix - will address in https://app.asana.com/0/0/1206170013724653/f